USBHelperLauncher/Net/ApplicationEndpoint.cs

```diff
@@ -55,5 +55,25 @@ public void GetZipProxy(Session oS)
             oS.responseBodyBytes = dataBytes;
             Proxy.LogRequest(oS, this, "Created zip from " + url);
         }
+
+        [Request("/requestZipHash.php")]
+        [Request("/hash_zip/*")]
+        public void GetZipHash(Session oS)
+        {
+            string url;
+            if (oS.PathAndQuery == "/requestZipHash.php")
```

Comment:
Could probably be shortened to something like:
```
string data = Path.GetFileName(oS.PathAndQuery);
if (data == "requestZipHash.php")
{
    data = GetRequestData(oS).Get("url");
}
byte[] bytes = Convert.FromBase64String(data);
string url = Encoding.UTF8.GetString(bytes);
```

Reply:
Oh, absolutely. I thought I had already changed that, this part of the code is just a remnant of me testing different edge cases.